assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image/frontend.tsx

```diff
@@ -78,5 +91,59 @@ interactivityStore(
 				},
 			},
 		},
+		effects: {
+			woocommerce: {
+				scrollInto: ( store: Store ) => {
+					// Scroll to the selected image with a smooth animation.
+					if (
+						store.selectors?.woocommerce?.isSelected( store ) &&
```

Comment:
`store.selectors?.woocommerce?.isSelected( store )` seems to be present in all the checks.

Can we perhaps handle it in a top-level condition?

Reply:
Good point! I fixed it with [8fe7233](https://github.com/woocommerce/woocommerce-blocks/pull/11113/commits/8fe72333fde771e7bca133d7cbf1b6e34ab481bb).